auth-api/src/auth_api/services/affiliation.py

```diff
@@ -457,7 +460,18 @@ async def get_affiliation_details(affiliations: List[AffiliationModel], org_id)
             )
 
         call_info = [
-            {"url": url, "payload": {"identifiers": identifiers}} for url, identifiers in url_identifiers.items()
+            {
+                "url": url,
+                "payload": {
+                    "identifiers": identifiers,
```

Comment:
could probably just serialize your data class here

Reply:
Let me see if I can find an example 

https://github.com/bcgov/sbc-pay/blob/04137958608dbde9fd70749760578ddbaf9052a0/pay-api/src/pay_api/utils/serializable.py#L14

But that's alot to bring over, I think what you have now is fine